host_vars/matterbridge.almalinux.org.yml

```diff
@@ -0,0 +1,160 @@
+---
+staging: false
+hashi_vault_path: "kv/data/infra/{% if staging %}stg/{% endif %}matterbridge"
+
+matterbridge_config:
+  general:
+    LogFile: /var/log/matterbridge/matterbridge.log
+  tengo:
+    RemoteNickFormat: /etc/matterbridge/protocol.tengo
+  mattermost.almalinux:
+    Server: chat.almalinux.org
+    Team: almalinux
+    Login: almarelay
+    Password: "{{ lookup('community.hashi_vault.hashi_vault', '{{ hashi_vault_path }}:mattermost_password',
+        token=lookup('env', 'VAULT_TOKEN'), url=secrets_url, errors='warn') | default('') }}"
+    RemoteNickFormat: "[{TENGO}] <{NICK}> "
+    PrefixMessagesWithNick: "true"
+    EditDisable: "true"
```

Comment:
People edit messages a good bit so I assume it might be a little annoying.  I set it here as-is based on our previous config.  Going to go ahead and merge this and we can revisit this setting separately if we want.